content/en/docs/languages/java/instrumentation.md

```diff
@@ -1185,9 +1185,10 @@ meter
 
 ## Logs
 
-Logs are distinct from Metrics and Tracing in that there is no user-facing logs
-API. Instead, there is tooling to bridge logs from existing popular log
-frameworks (e.g. SLF4j, JUL, Logback, Log4j) into the OpenTelemetry ecosystem.
+Logs are distinct from Metrics and Tracing in that **there is no user-facing
```

Comment:
this still feels weak to me because it obscures the message by other words. I would make this stand out:

> OpenTelemetry does not provide a Logging API. Instead, it integrates with existing log frameworks (e.g. SLF4j, JUL, Logback, Log4j) as a log exporter.

Also, a bit unrelated, but if feels weird that we only mention Java frameworks, maybe we could diversify?

Reply:
I like your wording.

>if feels weird that we only mention Java frameworks

This file is part of Java docs. We probably need disclaimers like this everywhere.

Reply:
```suggestion
Logs are distinct from metrics and traces in that **there is no user-facing
```

Since those are not proper nouns and the docs are not in German, no need to capitalize metrics and traces.

Also, please remove the **bold**, it should be use very sparingly, if at all.

Reply:
I would propose a wording like this:

```
(template)
OpenTelemetry does not provide a Logging API. Instead, it integrates with existing log frameworks as a log exporter.
(end template)

Example frameworks include:

* SLF4j
* JUL
* Logback
* Log4j
```